mountpoint-s3/CHANGELOG.md

```diff
@@ -1,6 +1,7 @@
 ## Unreleased (v1.21.0)
 
 * Change default logging level from WARN to INFO to improve visibility of important operational messages. ([#1605](https://github.com/awslabs/mountpoint-s3/pull/1605))
+* Change FUSE and S3 request metric names in logs (([#1630](https://github.com/awslabs/mountpoint-s3/pull/1630), [#1653](https://github.com/awslabs/mountpoint-s3/pull/1653))
```

Comment:
Thanks. Could you add the same entry to the `fs` crate and increment its version?